### PR TITLE
docs: add the `file-to-use` field

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Add the `file-to-use` field.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,4 +95,4 @@ jobs:
 ]
 ```
 
-From this repository usage https://github.com/EddieJaoudeCommunity/awesome-github-profiles
+From this repository usage <https://github.com/EddieJaoudeCommunity/awesome-github-profiles>
```
Comment (eddiejaoude) [inline README.md:98]: I think this is not needed

**Review by suitupstan — APPROVED** [review]
Eddie's suggestion, then 🚀

File: README.md (hunk 1 above)
Comment (Panquesito7) [reply README.md:98]: This is to fix Markdown linter issues. 🙂
I can still remove it if you want, though.

Comment (eddiejaoude) [reply README.md:98]: Probably best to have it as a raw link or use a markdown link but not inbetween, I will merge and another PR can fix this

**Review by eddiejaoude — APPROVED** [review]
Thanks

File: README.md (hunk 1 above)
Comment (Panquesito7) [reply README.md:98]: I'll submit a good first issue for that later. Thanks, Eddie!

Comment (eddiejaoude) [reply README.md:98]: thanks 👍

Comment (Panquesito7) [reply README.md:98]: Hey, Eddie! What should we use for this link?
Something like this:

From this repository usage: [Awesome GitHub profiles](https://github.com/EddieJaoudeCommunity/awesome-github-profiles)

Or should we use something else?

Comment (eddiejaoude) [reply README.md:98]: Looks good 👍

**Pull request merged**